api/python/quilt3/data_transfer.py

```diff
@@ -52,6 +52,15 @@
 MAX_FIX_HASH_RETRIES = 3
 
 
+def get_int_from_env(var_name):
+    val = os.getenv(var_name)
+    if val:
+        return int(val)
+
+
+MAX_CONCURRENCY = get_int_from_env('QUILT_TRANSFER_MAX_CONCURRENCY') or 10
```

Comment:
um this will fail for negative values, so we should check for the same in `get_int_from_env`